modules/migrations/migrate.go

```diff
@@ -161,12 +161,16 @@ func migrateRepository(downloader base.Downloader, uploader base.Uploader, opts
 		}
 
 		relBatchSize := uploader.MaxBatchInsertSize("release")
+		syncTags := false
 		for len(releases) > 0 {
-			if len(releases) < relBatchSize {
-				relBatchSize = len(releases)
+			if len(releases) <= relBatchSize {
+				if len(releases) < relBatchSize {
```

Comment:
For a clean code, I think this line is unnecessary. Even if `len(releases) == relBatchSize`, you can still `relBatchSize = len(releases)`.

Reply:
Ah, okay. I will look at this when I re-add the tag sync when there are no releases.